skimage/_shared/utils.py

```diff
@@ -226,7 +226,7 @@ def check_random_state(seed):
 
 
 def convert_to_float(image, preserve_range):
-    """Convert input image to double image with the appropriate range.
+    """Convert input image to float image with the appropriate range.
```

Comment:
This needs to say something like:

Floating point types will maintain their precision, while integer types will be converted to `double`.

Reply:
This also true for al the functions defined in `_warp_cy.pyx` and for `radon`. It is in my TODO list ;-)

Reply:
Sorry, I am not awake yet :-)
I will also fix the docstrings for `radon` and the function in `_warp_cy`.

Reply:
According to my [comment](https://github.com/scikit-image/scikit-image/pull/4128#discussion_r319407879), since not all the floating point precision are supported, I added a **Notes** section in the docstring specifying this: _"Input images with `float32` data type are not upcasted."_